azkaban-hadoop-security-plugin/src/main/java/azkaban/security/AbstractHadoopSecurityManager.java

```diff
@@ -528,7 +528,7 @@ private void assignPermissions(final String user, final File tokenFile, final St
   protected String kerberosSuffix(final Props props) {
     // AZKABAN_SERVER_HOST_NAME is not set in Props here, get it from another instance of Props.
     final String host = ServiceProvider.SERVICE_PROVIDER.getInstance(Props.class)
-        .getString(AZKABAN_SERVER_HOST_NAME, "unknown");
+        .getString(AZKABAN_WEBSERVER_EXTERNAL_HOSTNAME, "unknown");
```

Comment:
This will change things for both on prem and containerization. Is that ok?

Reply:
Yes. This way, the grid attribution pipeline does not have to handle things differently keeping it simple

Reply:
The comment above this line is still referring to `AZKABAN_SERVER_HOST_NAME`. Please fix that and update to include a note about why we are choosing the webserver name as part of the ugi.

Reply:
Done